Code/C++/inverse_of_an_array.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+INVERSE OF AN ARRAY
+
+If the array elements are swapped with their corresponding indices,
+the array finally results is inverse of an array.
+
+For Example,
+If we have 4,0,2,3,1 in original array with indices 0,1,2,3,4 respectively,
+so assume a blank array of same size. Now at 4th index, element in inverse array will be 0.
+Similarly, at 0th index, element in inverse array will be 1
+and similar swapping for the rest of elemnts
+At last you will be getting 1,4,2,3,0 in inverse array.
+
+QUESTION-->
+
+Input:
+1)First line contains t, number of test cases.
+2)Second line contains size of the original array.
+3)Third line contains elements of the array.
+
+Output:
+1)contains t lines of output, inverse array of each original array. 
+
+Sample input:

```

Comment:
Hey, plz do not add sample test cases and complexity here.  Only provide a brief description of problem statement here.